lib/cdo/help_header.rb

```diff
@@ -59,10 +59,9 @@ def self.get_help_contents(options)
     end
 
     if options[:level] && options[:level].game == Game.music
-      options[:request].path.starts_with?('/projectbeats')
```

Comment:
@breville  Just want to confirm that this is right? Or should it be changed to /music?

Reply:
I suspect there was a missing `||` because I don't see how this line does anything!  I think it's okay to remove it completely since we'll be redirecting users from `/projectbeats` to a new music project.